pkg/workers/push/push.go

```diff
@@ -149,11 +149,14 @@ func pushToAndroid(ctx *jobs.WorkerContext, msg *Message) error {
 		To:       msg.DeviceToken,
 		Priority: priority,
 		Notification: &fcm.Notification{
-			Body:  msg.Message,
-			Title: msg.Title,
 			Sound: msg.Sound,
 		},
-		Data: map[string]interface{}{"topic": msg.Topic},
+		Data: map[string]interface{}{
+			"topic":             msg.Topic,
+			"content-available": 1,
```

Comment:
can we put something like true instead of 1 ?

Reply:
I just tested, the answer is yes. You want me to do that ?